src/liquid/components/ld-sidenav/ld-sidenav-back/ld-sidenav-back.tsx

```diff
@@ -0,0 +1,124 @@
+import '../../../components' // type definitions for type checks and intelliSense
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Listen,
+  Method,
+  Prop,
+  State,
+} from '@stencil/core'
+import { getClassNames } from '../../../utils/getClassNames'
+import { closest } from '../../../utils/closest'
+
+/**
+ * @virtualProp ref - reference to component
+ * @virtualProp {string | number} key - for tracking the node's identity when working with lists
+ */
+@Component({
+  tag: 'ld-sidenav-back',
+  styleUrl: 'ld-sidenav-back.css',
+  shadow: true,
+})
+export class LdSidenavBack {
+  @Element() el: HTMLElement
+  private sidenav: HTMLLdSidenavElement
+
+  /** Emitted on click. */
+  @Event() ldSidenavBack: EventEmitter
+
+  /** Used as aria-label for the back button */
+  @Prop() backLabel = 'Back'
+
+  @State() parentLabel = ''
+  @State() rounded = false
+  @State() sidenavCollapsed: boolean
+  @State() sidenavClosable: boolean
+
+  @Listen('ldSidenavCollapsedChange', { target: 'window', passive: true })
+  handleSidenavCollapsedChange(ev) {
+    if (ev.target !== this.sidenav) return
+    this.sidenavCollapsed = ev.detail
+  }
+
+  @Listen('ldSidenavBreakpointChange', { target: 'window', passive: true })
+  handleSidenavBreakpointChange(ev) {
+    if (ev.target !== this.sidenav) return
+    this.sidenavClosable = ev.detail
+  }
+
+  /**
+   * @internal
+   * Updates the label of the back button.
+   */
+  @Method()
+  async updateLabel(text?: string) {
+    this.parentLabel = text || ''
+  }
+
+  private onClick = () => {
+    this.ldSidenavBack.emit()
+  }
+
+  private onKeyDown = (ev) => {
+    if ([' ', 'Enter'].includes(ev.key)) {
+      ev.preventDefault()
+      this.ldSidenavBack.emit()
+    }
+  }
+
+  componentWillLoad() {
+    this.sidenav = closest('ld-sidenav', this.el)
+    this.rounded = !!this.el.querySelector('ld-sidenav-navitem[rounded]')
```

Comment:
> Why not make this an explicit prop instead if "inheriting" it from the slotted element?

There is only one component which has this prop -> less confusion, less ways to mess things up: If you have a rounded navitem, the button is round, if not, it's not. Simple.

> It might even be possible, that a subnav consists of rounded navitems, while the 1st level nav doesn't. Using a prop, the dev would have the chance to reflect that.

Subnavs or navitems within subnavs have no relationship with the navitem inside the back button in a sense that they need to share a common look.
Also, if you would allow for different states, you would end up with weird hops when transitioning back, because the element has hover state after click on the back button and also during the transition.